roles/etcd_client_certificates/meta/main.yml

```diff
@@ -14,3 +14,4 @@ galaxy_info:
   - system
 dependencies:
 - role: etcd_common
+- role: delegated_serial_command
```

Comment:
It'd be nice to minimize the dependency on master and node roles, we should pass in any master/node specific values to this role instead, I think. etcd_ca seems like a hard dependency to me, though.